util/crc32c.cc

```diff
@@ -404,5 +542,12 @@ uint32_t Extend(uint32_t crc, const char* buf, size_t size) {
   return ChosenExtend(crc, buf, size);
 }
 
+// We want to check if this cpu feature is available at runtime, 
+// *not at compile time*. 
+static inline bool isRuntimeSSE42Available() {
```

Comment:
this function is not used.

Reply:
Well, that's a functionality added. We can used it to check for a specific feature at runtime. That's why it is ther.e